lib/pos_fee_estimator.go

```diff
@@ -290,7 +290,7 @@ func (posFeeEstimator *PoSFeeEstimator) pruneBlocksToMaxNumPastBlocks(blocks []*
 // EstimateFeeRateNanosPerKB estimates the fee rate in nanos per KB for the current mempool
 // and past blocks using the congestionFactorBasisPoints, priorityPercentileBasisPoints, and
 // maxBlockSize params.
-func (posFeeEstimator *PoSFeeEstimator) EstimateFeeRateNanosPerKB(minFeeRateNanosPerKB uint64, maxBlockSize uint64) uint64 {
+func (posFeeEstimator *PoSFeeEstimator) EstimateFeeRateNanosPerKB(minFeeRateNanosPerKB uint64) uint64 {
```

Comment:
The fee estimator already has access to the global params and the soft max block size. That param doesn't need to be passed in by the caller.